### PR TITLE
Create a site to display horror shows

### PR DESCRIPTION
**Pull request by ijeomaodinko**
- This file used webpack, used ES6 syntax and modules, and follow Gitflow
- Created functions get shows from an API, getShows, creatItem, cleanElement, showListItem. 
- Added section to display shows in index.html file. 
- Created function postLike to add a new like for a specific item.
- Created function getLikes to get the likes from API.
- Created function itemsCounter to show the number of shows in the nav.
- Created function post and get comments per item. Also function that shows attributes of an item in a modal. 
- Linters files installed.
- You can watch a video about our work here
https://drive.google.com/file/d/1shboTJuRBJfrZyveVo3mJLUYlUnOsuCx/view?usp=sharing

### CODE REVIEW

File: src/modules/addPopup.js [hunk 1]
```diff
@@ -1,118 +1,130 @@
...
+  // Add content and atributes
+  divShowModal.classList.add('modal');
+
+  imgShowModal.src = show.image.medium;
+  imgClose.classList.add('closeModal');
+
+  // Show image
+  divImgModal.classList.add('imageModal');
+  imgClose.src = cancel;
+  divImgModal.appendChild(imgShowModal);
```
Comment (Reem-lab) [inline src/modules/addPopup.js:43]: - [ ] Great job working on the pop up but as I can see the information is a summary and i think you should put four things displaying some information for each movie ✅


- To understand me better I am talking about this part 
image


- Microverse design
image

File: README.md [hunk 2]
```diff
@@ -1,37 +1,104 @@
...
+
+    npm run build
+
+
+### Run Tests
+
+To run test on the test files used:
+
+    npm test 
+
```
Comment (Reem-lab) [inline README.md:63]: - [ ] I think the tests are not working so kindly fix the tests to passed and make more tests to your project 🙏🏻✅✅

File: README.md [hunk 3]
```diff
@@ -1,37 +1,104 @@
...
+- Webpack
 
-# Header 1
-## Header 2
-### Header 3
+## Live Demo (if available)
 
-- Bulleted
-- List
+[Live Demo Link](https://livedemo.com)
```
Comment (Reem-lab) [inline README.md:18]: - [ ] I think you forget to deploy your project so kindly make sure to deploy it since it is one of the requirements ✅✅🙏🏻🙏🏻

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻 

Good job so far!🔥🔥🔥🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights ✨✨
📌 No Linters error ✅✅
📌 Good commit messages✅✅
📌 Descriptive pull request ✅✅
📌 Great working on the Javascript functionality ✅✅
📌 Professional README file ✅✅


## Required Changes ♻️
_Check the comments under the review._

## [ REQUIRED ] You followed the Gitflow Great job but for the project you should make a pull reuest from the dev to the main so kindly merge this branch and make a pull request from the dev to the main ✅✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addPopup.js (hunk 1 above)
Comment (VaneCode) [reply src/modules/addPopup.js:43]: Hello @Reem-lab, thanks for your feedback. I have one doubt we have to use the new pull request against the main to ask for a new code review or do we have to use this one?
Thanks.

Comment (Reem-lab) [reply src/modules/addPopup.js:46]: Hello @VaneCode Yes you have to merge this branch to dev then you can make a new pull request from dev to main then send the pull request to ask for a review 🙏🏻🙏🏻🙏🏻✅✅

**Pull request merged**